lib/windspharm/tools.py

```diff
@@ -43,7 +43,7 @@ def __order_dims(d, inorder):
 
 
 def __reshape(d):
-    out = d.reshape(d.shape[:2] + (np.prod(d.shape[2:]),))
+    out = d.reshape(d.shape[:2] + (np.prod(d.shape[2:], None, np.int),))
```

Comment:
Can you change it to `np.prod(d.shape[2:], dtype=np.int)` please? Using the keyword argument explicitly makes the intent much more obvious when reading it.